js/visearch.js

```diff
@@ -86,6 +90,25 @@ const RESULT_LOAD = 'result_load';
       }
     }
 
+    function sendEvents(action, events, callback = () => { }, failure = () => { }) {
+      tracker = getTracker();
+
+      if (tracker && tracker.validateEvents(events, failure)) {
+        const batchId = tracker.generateUUID();
+
+        events.forEach((event) => {
+          event.v = SDK_VERSION;
+          event.sdk = SDK;
+
+          if (action.toLowerCase() === 'transaction' && !event.transId) {
```

Comment:
This means that all scripts (widget/deploy script/banner) should also support non case sensitive transaction events right?

Reply:
ideally yes.

Reply:
same goes for add_to_cart as well

Reply:
in our data pipeline we will also auto convert all actions to lowercase

Reply:
Does this also apply for `add_to_cart`?